src/features/TodoGroup.js

```diff
@@ -0,0 +1,8 @@
+import React from "react";
+import TodoItem from "./TodoItem";
+
+export default function TodoGroup(props) {
+    return props.items.map((item) => {
+        return <TodoItem item={item} />;
```

Comment:
can add key for todoItem
![image](https://user-images.githubusercontent.com/117631445/207205065-73417875-5569-4745-bb10-2152fed9fdb0.png)